README.md

```diff
@@ -165,3 +165,17 @@ This can be useful when most modules from a repository are trusted and few modul
 ```
 
 This config will install all modules except ``untrusted_module_1`` and ``untrusted_module_2``.
+
+**Include Only Specific Languages**
```

Comment:
Codacy found an issue: [[no-emphasis-as-heading] Don’t use emphasis to introduce a section, use a heading](https://app.codacy.com/gh/Numigi/gitoo/pullRequest?prid=6159859)